### PR TITLE
test: improve rbd-mirror test case coverage

### PR DESCRIPTION
**Pull request by dillaman**

### CODE REVIEW

File: qa/workunits/rbd/rbd_mirror_stress.sh [hunk 1]
```diff
@@ -1,412 +1,149 @@
 #!/bin/bash
```
Comment (trociny) [inline qa/workunits/rbd/rbd_mirror_stress.sh]: Minor: it looks like you have removed dependency on bash $RANDOM, so may be change this to /bin/sh?

File: src/vstart.sh [hunk 2]
```diff
@@ -135,6 +136,7 @@ usage=$usage"\t--bluestore use bluestore as the osd objectstore backend\n"
 usage=$usage"\t--memstore use memstore as the osd objectstore backend\n"
 usage=$usage"\t--cache <pool>: enable cache tiering on pool\n"
 usage=$usage"\t--short: short object names only; necessary for ext4 dev\n"
+usage=$usage"\t--no_lockdep disable lockdep\n"
```
Comment (trociny) [inline src/vstart.sh]: minor: (just ignore if you don't agree) To be consistent with other option names ('--nodaemon)', I would prefer if it were '--nolockdep' instead.

**Comment by dillaman** [comment]
@trociny updated

**Comment by trociny** [comment]
@dillaman I have just merged #9618, could you please rebase?

**Comment by dillaman** [comment]
@trociny rebased

**Pull request merged**